### PR TITLE
chore: IaC PE output

### PR DESCRIPTION
**Pull request by YairZ101**
#### What does this PR do?
This PR adds the ability to output the `snyk-iac-test` scan results into the CLI.

Currently, the output is missing the following:
1. In the issue description, the `Info`, `Rule`, and `Resolve` fields.
2. Test summary.
3. Error output.

Each of the above will be added in future PRs.

#### How should this be manually tested?
1. Run `snyk-dev iac test --experimental`
2. Experience our glorious CLI output

#### Any background context you want to provide?
This PR removes the single acceptance test that we had for the `experimental` flag, the reason behind that is that we understood that testing the test flow from E2E without downloading the `snyk-iac-test` binary and the Snyk rules bundle is not an easy task.
Instead, I've added a unit test for the `test` function that checks that we return the expected response.
In addition, the team will look into adding E2E tests that will run as part of the env. testing tool.

#### What are the relevant tickets?
https://snyksec.atlassian.net/browse/CFG-1887

#### Screenshots
image

### CODE REVIEW

File: test/jest/unit/cli/commands/test/iac/v2/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import * as fs from 'fs';
```
Comment (JackuB) [inline test/jest/unit/cli/commands/test/iac/v2/index.spec.ts:1]: I think you want to add this file (or folder?) to the codeowners file as owned by IaC:
Screen Shot 2022-06-30 at 19 30 24

File: src/lib/formatters/iac-output/v2/issues-list/formatters.ts [hunk 2]
```diff
@@ -27,3 +32,49 @@ export function formatScanResultsNewOutput(
...
+export function formatSnykIacTestScanResultNewOutput(
+  snykIacTestScanResult: Results | undefined,
+): IacTestOutput {
+  const formattedResults: IacTestOutput = {
+    results: {},
+    // TODO: Add metadata when working on the Share Results feat
+    metadata: { projectName: 'TBD', orgName: 'TBD' },
+  };
+
+  if (snykIacTestScanResult) {
```
[the remark this answers is not in the document]
Comment (francescomari) [reply src/lib/formatters/iac-output/v2/issues-list/formatters.ts]: Fixed at dbef08e0.

File: test/jest/unit/cli/commands/test/iac/v2/index.spec.ts (hunk 1 above)
Comment (francescomari) [reply test/jest/unit/cli/commands/test/iac/v2/index.spec.ts:1]: Fixed at dbef08e0.

**Review by robcresswell — APPROVED** [review]

**Comment by robcresswell** [comment]
A +1500 chore? This doesn't look like a chore pr

**Review by JackuB — APPROVED** [review]
Approving the codeowners file change

**Comment by francescomari** [comment]
@robcresswell yes, it should probably be a feat, but it's part of a bigger feature still in development and hidden behind command line flags and feature flags. We will keep it in mind for the next ones.

File: src/lib/formatters/iac-output/v2/issues-list/formatters.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (YairZ101) [reply src/lib/formatters/iac-output/v2/issues-list/formatters.ts]: nice catch @ipapast :)
@francescomari my assumption was that `snyk-iac-test` will not print `results` if there aren't any (similar to how we don't print `errors` if there aren't any), was it intentional that we will always print `results`?

**Pull request merged**